README.md

```diff
@@ -1 +1,288 @@
-# amar-hello-microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [ ] To ensure your project is professional and user-friendly, it's important to make sure that all links and images are working properly. In regards to the image in question, you can either replace it with your own logo or remove it completely. 💯

![image](https://user-images.githubusercontent.com/107952724/234680370-24555e6c-9afe-4772-b7f4-74c9f5d25eb6.png)